web-ui/src/components/received_request_card/ReceivedRequestCard.jsx

```diff
@@ -21,21 +22,21 @@ const classes = {
 };
 
 const StyledCard = styled(Card)({
-  [`& .${classes.redTypography}`]: {
-    color: '#FF0000'
-  },
-  [`& .${classes.yellowTypography}`]: {
-    color: '#EE8C00'
-  },
-  [`& .${classes.greenTypography}`]: {
-    color: '#006400'
-  },
-  [`& .${classes.darkGrayTypography}`]: {
-    color: '#333333'
-  },
-  [`& .${classes.grayTypography}`]: {
-    color: 'gray'
-  }
+  // [`& .${classes.redTypography}`]: {
+  //   color: '#FF0000'
+  // },
+  // [`& .${classes.yellowTypography}`]: {
+  //   color: '#EE8C00'
+  // },
+  // [`& .${classes.greenTypography}`]: {
+  //   color: '#006400'
+  // },
+  // [`& .${classes.darkGrayTypography}`]: {
+  //   color: '#333333'
+  // },
+  // [`& .${classes.grayTypography}`]: {
+  //   color: 'gray'
+  // }
```

Comment:
Same question [mark had](https://github.com/objectcomputing/check-ins/pull/2334/files#r1592538204). If we want to keep the styles for now, could we add a comment in the code as to when they're safe to remove?

Reply:
I think I just missed removing this, will do in a moment.